airbyte-webapp/src/components/base/Switch/Switch.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import styles from "./Switch.module.scss";
+
+interface SwitchProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  small?: boolean;
+  loading?: boolean;
+}
+
+export const Switch: React.FC<SwitchProps> = ({ loading, small, checked, value, ...props }) => {
+  const isChecked = checked || !!value;
+  const labelStyle = `${styles.switch} ${small ? styles.small : ""} ${loading ? styles.loading : ""}`;
+  const spanStyle = `${styles.slider} ${small ? styles.small : ""} ${loading ? styles.loading : ""} ${
+    isChecked ? styles.checked : styles.unchecked
+  }`;
```

Comment:
We may want to use [classnames](https://github.com/JedWatson/classnames)
I'm always hesitant to bring in a new library for simple functionality however.

Reply:
We should def use class names imho. Depending on the component the classes we need to set might be way more complex than here and I don't think template strings will be a good solution once you have more classnames or conditionals. Given that the library is also extremely wide spread and used I wouldn't worry about having it as a dependency

Reply:
I believe we discussed/approved bringing in `classnames` in team time last week!  Sounds good to me.  It's helpful.